README.md

```diff
@@ -1,2 +1,16 @@
 # ck-nntest-outputs
-Reference outputs for nntest
+Reference outputs for `nntest`
+
+`*-tensorflow-cpu` programs were used as reference source when possible or `*-armcl-opencl` otherwise.
+
+Example command to run a program with validation:
+
+```bash
+ck run program:conv-tensorflow-cpu \
```

Comment:
Is it `ck run program:*` or `ck run nntest:*`?

Reply:
`ck run program:*` is enough for validation. `nntest` does much more work, so I've decided to give a simpler program as an example.